llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -8858,13 +8794,64 @@ static void addCanonicalIVRecipes(VPlan &Plan, Type *IdxTy, bool HasNUW,
                        {CanonicalIVIncrement, &Plan.getVectorTripCount()}, DL);
 }
 
+static VPValue *addResumeValuesForInduction(VPHeaderPHIRecipe *PhiR,
+                                            VPBuilder &Builder,
```

Comment:
```suggestion
                                            VPBuilder &VectorPHBuilder,
```
consistent with `ScalarPHBuilder`.